packages/web-react/src/utils/common.ts

```diff
@@ -0,0 +1,4 @@
+export const getRandomId = (length = 5) =>
```

Comment:
I think it is up to the user to set the ID attribute. And also there are better generators like [nanoid](https://www.npmjs.com/package/nanoid). What kind of problem are you solving with this approach?

Reply:
My point was to not have to import another library for such banality. In addition, we already use a similar script with 'Dropdown', so I decided to turn it into a utility. And the user doesn't have to give the id every time, only when he needs to. Otherwise, it will therefore be generated automatically and the dialogue with the caption will be bound.

Reply:
I was not suggesting adding another library. My point of view is that the `id` should be mandatory and a developer should decide how the `id` should be generated or be static. In this case, it can have a performance side effect on unwanted rerenders. This function generates a random id even if there is no other change in other props, so I believe it causes rerender of the entire component every time.

Also, think that this is not a good idea in the Dropdown and we should remove it either.

I did a quick research and did not find a component library which is generating id for a developer.

I think that the intention is good to have a nice and easy API for the component but the harder I try to grab that idea the more I think that we should not do it this way.

Reply:
We agreed to remove this function and make the `id` prop mandatory.